pyproject.toml

```
@@ -25,6 +25,7 @@ dev = [
     "pytest>=8.4.0",
     "pytest-cov~=6.0",
     "pytest-mock~=3.12",
+    "sphinx-copybutton",
```

Let's pin this to a minor version.


```suggestion
    "sphinx-copybutton>=0.5",
```